src/utils/dataset_utils.py

```diff
@@ -903,11 +853,6 @@ def create_candidate_class(extraction_type):
         LocationExtraction = candidate_subclass('Location', ['location'])
         candidate_class = LocationExtraction
         candidate_class_name = 'LocationExtraction'
```

Comment:
Need to add phone bit in here -- see diff

Reply:
Done